odoo/sql_db.py

```diff
@@ -806,6 +808,7 @@ def connection_info_for(db_or_uri):
     :param str db_or_uri: database name or postgres dsn
     :rtype: (str, dict)
     """
+    app_name = "odoo_pid%d" % os.getpid()
```

Comment:
`pid` infix doesn't seem very useful, or to help with readability.

Reply:
The `pid` is helpful to match the logs

The crons will show the following messages:

    ... 757538 INFO db odoo.addons.base.models.ir_cron: Starting job `Name of the cron`.
    ... 757538 INFO odoo odoo.sql_db: Cursor created backend pid 394452

Notice the PID of Odoo `757538` and the postgresql pid `394452`

So, the postgresql.log will show the following log:

    [394452]: [371-1] db=odoo,user=odoo,client=127.0.0.1,app=odoo_pid757538 LOG:  00000: duration: 0.074 ms  statement: QUERY

Matching the `odoo_pid757538` and the postgresql backend pid.

It is helpful to know what queries was executed from backend_pid 394452 and odoo pid

Or are you requesting the following change?

```suggestion
    app_name = "pid%d" % os.getpid()
```

Reply:
> Or did you mean... you are requesting the following change?

Yes, I wasn't saying the pid is not useful (I completely agree it is), just that the actual "pid" part of the output seems to just add noise.

I actually would have preferred `odoo_%d` (or `odoo-%d`) which I think nicely identifies the odoo instance from the postgres side, which I understand is the point.

Reply:
I got it, thank you!